mmv1/templates/terraform/examples/cloudbuild_trigger_service_account.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
+  trigger_template {
+    branch_name = "master"
+    repo_name   = "my-repo"
+  }
+
+  service_account = "projects/my-project/serviceAccounts/my-service-account"
+  filename        = "cloudbuild.yaml"
+}
```

Comment:
I expect you'll want this resource to get provisioned only once the service account has the right permissions, too. There's no "natural" value to reference to establish a dependency-- instead, let's use https://www.terraform.io/docs/language/meta-arguments/depends_on.html to reference both `*_iam_member` resources

Reply:
Done.